readme.md

```diff
@@ -13,13 +13,18 @@ cp .env.example .env
 ```shell script
 docker-compose up -d
 ```
+### 3.Install frontend dependencies and build frontend:
+```
+docker-compose exec node npm install
+docker-compose exec node npm run build
+```
 
-### 3. Install backend dependencies and build static API endpoints:
+### 4. Install backend dependencies and build static API endpoints:
 ```shell script
 docker-compose exec php composer install
 docker-compose exec php composer build
 ```
-
+Application should be available under `localhost:8037` or other port if you changed `EXTERNAL_WEBSERVER_PORT` value in `.env` file.
```

Comment:
Under localhost:8037 the application is not available, but under localhost:8027 it is.

Reply:
Okay, fixed the typo in readme.md , thanks!